integreat_compass/__init__.py

```diff
@@ -1 +1,4 @@
+"""
+Shows current version of Integreatcompass
+"""
 __version__ = "2023.3.0"
```

Comment:
I think it's a bit misleading to reduce the docstring of the __init__ file to only the version, it is much more the entry point for the whole integreat compass package.

```suggestion
This package contains the Integreat Compass, a digital platform aimed at improving the integration of refugees and migrants into German society and the labor market 
"""

#: The current version of integreat-compass
__version__ = "2023.3.0"
```